### DISCUSSION TITLE
Daily 28/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#174] [Daily Sprint]
- Qué hice ayer?
- Que haré hoy?
- Dificultades o impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve haciendo en Angular el formulario de Login. Se me presentan dudas a medida que lo voy haciendo, así que seguí investigando un poco mas, con lo cual no lo pude terminar. Por la tarde tuve clases de programación del Ispc.
Hoy continúo con el login. Por la tarde tengo turno médico y eso me va a quitar tiempo. De todas maneras aspiro a terminar todas las tareas que tengo asignadas este fin de semana para poder trabajar en lo que quede pendiente la semana que viene.

**Comment by rpgrca** [comment]
Hoy no pude hacer nada, bastante ocupado. Mañana es posible que tenga tiempo libre así que voy a intentar hacer las tareas necesarias como para tener el sitio andando. También voy a ver qué más del back se necesita.

**Comment by BraianTroncoso** [comment]
Hoy voy asignarme algunas tareas y ver cual puedo hacer, ya que por trabajar de manera mala hice tareas ya realizadas y por ende fue una perdida de tiempo. Espero poder realizar algunas.

**Comment by Brian-Almada** [comment]
Ayer pude armar el esqueleto de catálogo. Hoy intentando armar las tarjetas que contienen los productos y la información del producto.

**Comment by BraianTroncoso** [comment]
Estuve viendo el repositorio, lo cloné otra vez porque no ví cambios soloamente nuevas ramas creadas, me moví a esas ramas y tampoco logré ver cambios. Digo esto, porque queria asignarme alguna Issue de acuerdo a componentes ya finalizados como los que estaban cerrados para seguir de ahi, y ver un poco más(porque no estaria entendiendo bien) pero no encontré nada, debe ser un error o algo que estoy ignorando.
